README.md

```diff
@@ -1 +1,5 @@
-# harmony
\ No newline at end of file
+# harmony
+
+
+
```

Comment:
Add a line that explains how to start up the dgraph instance. Ultimately, we'll want to create a startup script that initializes both the application and the data backends using the right environments.